tests/testthat/test-lsgameofchance.R

```diff
@@ -21,48 +15,39 @@
   test_that("Game of Chance two-player plot matches", {
     plotName <- results[["results"]][["CIPlot"]][["data"]]
     testPlot <- results[["state"]][["figures"]][[plotName]][["obj"]]
-    jaspTools::expect_equal_plots(testPlot, "two-player CIPlot", dir = "LSgameofchance")
+    jaspTools::expect_equal_plots(testPlot, "probability-of-player-1-winning-two-players", dir = "LSgameofchance")
```

Comment:
given that we now have the test framework updated you can delete the dir argument
```suggestion
    jaspTools::expect_equal_plots(testPlot, "probability-of-player-1-winning-two-players")
```